lib/project_types/node/messages/messages.rb

```diff
@@ -202,7 +202,10 @@ module Messages
             host_must_be_https: "HOST must be a HTTPS url.",
           },
 
-          open_info: "{{*}} Press {{yellow: Control-T}} to open this project in {{green:%s}} ",
+          open_info: <<~MESSAGE,
+            {{*}} To view your app, open the following URL in your browser:
+            {{green:%s}}
+          MESSAGE
```

Comment:
Why not add the newlines (blank lines) before and after the `open_info` message in the `messages.rb` file and remove the `"\n"` from the `ctx.puts` in the `serve.rb` files?

Reply:
That's a great question, and that's the way I implemented it at first. But then I started thinking if those `\n`s are part of the message or of the layout I'm going for in the CLI.

Reply:
Also, this one is more for @gfscott: we already have a 'generic' way of printing out a URL, which says `Please open this URL in your browser: ...`. I was also torn if it would be better to use that here and keep it consistent or if this scenario called for a little more context.

Reply:
I think the additional context is valuable in this case! It's not just opening a URL, it's a required step to complete the OAuth flow. 

We could even up the detail level and make it a little more conversational:
```
{{*}} To install and start using your app, open this URL in your browser:
```